src/components/organisms/NavBarSchedule/NavBarSchedule.scss

```diff
@@ -126,3 +130,28 @@ $schedule-btn--height: 51px;
     padding: $spacing--sm 0px;
   }
 }
+
+@media screen and (max-width: $media-breakpoint--md) {
+  .NavBarSchedule {
+    width: 100%;
+  }
+
+  .NavBarWrapper {
+    &__download-buttons {
+      width: 100%;
+    }
+
+    &__close-button {
+      height: 45px;
+      width: 45px;
+      display: block;
+      position: absolute;
+      right: $spacing--md;
+      z-index: $z-layer-navbar;
```

Comment:
You should use the `z` function from `constants.scss` and the value from `$z-layers`

Reply:
fixed in 5afa7cde5b6bd9d643aeb30b9b72287e5fc9bb5b